### ISSUE TITLE
UniversalFeedClient.ListPackagesAsync does not filter on groups

### ISSUE THREAD

**Issue by wldevries** [Inedo/Inedo.UPack#8]
**Summary**
I'm trying to list the packages on our Proget server (free version) for a specific group using the following method:
```
var client = new Inedo.UPack.Net.UniversalFeedClient(ProgetFeed);
var packs = await client.ListPackagesAsync("myGroup", null);
```
**Expected result**
The packs variable contains all packages on the server under the "myGroup" group.

**Actual result**
The packs variable contains all packages on the server regardless of the group.

**Versions**
Inedo.UPack nuget 1.0.0
Proget 5.0.10

**Comment by wldevries** [comment]
> PG-1258 - FIX: Universal feeds with connectors cannot be searched by group
Possibly fixed in a later version of Proget.

**Comment by BenLubar** [comment]
This was fixed in ProGet 5.0.12: [PG-1258](https://inedo.myjetbrains.com/youtrack/issue/PG-1258)
